src/prefect_server/api/flows.py

```diff
@@ -208,7 +210,8 @@ async def create_flow(
         name=flow.name,
         serialized_flow=serialized_flow,
         environment=flow.environment,
-        core_version=flow.environment.get("__version__"),
+        run_config=flow.run_config,
+        core_version=flow.__version__,
```

Comment:
I think we need `flow.__version__ or flow.environment.get("__version__")` for backwards compatibility

Reply:
All flows have a `__version__` attached already (and have for a while), it's part of the base `ObjectSchema` serializer. I don't think a backwards compat fix is needed here, but might be wrong.

Reply:
Ahh you're right